bots/zephyr_mirror_backend.py

```diff
@@ -438,6 +451,7 @@ def process_notice(notice, log):
             os._exit(0)
 
 def decode_unicode_byte_strings(zeph):
+    # type: (Dict[str, Any]) -> Dict[str, Any]
```

Comment:
line 434 zeph = decode_unicode_byte_strings(zeph) -> zeph is of type dict[str, str]